src/app/story/directives/metrics-downloads.component.spec.ts

```diff
@@ -8,41 +8,30 @@ describe('MetricsDownloadsComponent', () => {
 
   provide(TabService);
 
-  beforeEach(() => {
-    castle.mockList('prx:episode-downloads', [{
-      downloads: [
-        [
-          '2017-04-19T00:00:00Z',
-          99
-        ],
-        [
-          '2017-04-20T00:00:00Z',
-          97
-        ]
-      ]
-    }]);
-  });
+  const mockDownloads = [{
+    downloads: [
+      ['2017-04-19T00:00:00Z', 99],
+      ['2017-04-20T00:00:00Z', 97]
+    ]
+  }];
 
   cit('should not make castle request if episode not published', (fix, el, comp) => {
-    spyOn(castle, 'followList').and.stub();
```

Comment:
This method no longer exists on the global testing `cms` var (which is renamed to `castle` here).  You can't directly access the root api haldoc.

But ... by not mocking anything here, we can be sure we'd get an "unmocked request" error if the component did try to follow a link.